### PR TITLE
Handle empty i_stream in chunked uploader start call.

### PR DESCRIPTION
**Pull request by simi**
Hello.

I have found minor issue using chunked uploader.

# How to reproduce

```ruby
# reproduce_problem.rb

require 'dropbox_api'

client = DropboxApi::Client.new(token)
content = ""
client.upload_by_chunks('/test.txt', content, mode: :add, autorename: true)
```

```bash
[retro@retro  dropbox_api (master %=)]❤ ruby reproduce_problem.rb 
Traceback (most recent call last):
        4: from reproduce_problem.rb:7:in `<main>'
        3: from /home/retro/.gem/ruby/2.5.1/gems/dropbox_api-0.1.12/lib/dropbox_api/endpoints/virtual/upload_by_chunks.rb:44:in `upload_by_chunks'
        2: from /home/retro/.gem/ruby/2.5.1/gems/dropbox_api-0.1.12/lib/dropbox_api/chunked_uploader.rb:17:in `start'
        1: from /home/retro/.gem/ruby/2.5.1/gems/dropbox_api-0.1.12/lib/dropbox_api/client.rb:14:in `block in add_endpoint'
/home/retro/.gem/ruby/2.5.1/gems/dropbox_api-0.1.12/lib/dropbox_api/endpoints/files/upload_session_start.rb:34:in `block in <class:UploadSessionStart>': undefined method `bytesize' for nil:NilClass (NoMethodError)
```

# TL;DR; reason

```ruby
chunk_size = 4 * 1024 * 1024 # default value, anyway anything >= 1 will fail the same way
StringIO.new("").read(chunk_size).bytesize #=> NoMethodError (undefined method `bytesize' for nil:NilClass)
StringIO.new("test").read(chunk_size).bytesize #=> 4
```

Even first reading of IO can be the end (resulting into `nil`) which is not expected at https://github.com/Jesus/dropbox_api/blob/42b597f8995011afaf7b531c3e5bed82c29de77e/lib/dropbox_api/chunked_uploader.rb#L15.

# Detailed reason

1. empty string is casted to `StringIO` and passed to chunked uploader at https://github.com/Jesus/dropbox_api/blob/42b597f8995011afaf7b531c3e5bed82c29de77e/lib/dropbox_api/endpoints/virtual/upload_by_chunks.rb#L41-L44

2. chunk size is initialized to 4MB at https://github.com/Jesus/dropbox_api/blob/42b597f8995011afaf7b531c3e5bed82c29de77e/lib/dropbox_api/chunked_uploader.rb#L6

3. reading chunk from empty `StringIO` (resulting into `nil`) and passing it into `upload_session_start` happens at https://github.com/Jesus/dropbox_api/blob/42b597f8995011afaf7b531c3e5bed82c29de77e/lib/dropbox_api/chunked_uploader.rb#L15-L17

4. final `bytesize` call at `nil` is done at https://github.com/Jesus/dropbox_api/blob/42b597f8995011afaf7b531c3e5bed82c29de77e/lib/dropbox_api/endpoints/files/upload_session_start.rb#L34

# Summary

This is a naive fix, which passes an empty string when the chunk is empty.
As a result an empty file is created in the desired Dropbox account.

PS: With this patch `upload_by_chunks` and `upload` works the same way with empty content.

### CODE REVIEW

**Comment by Jesus** [comment]
Super thanks for the clear explanation.

**Comment by simi** [comment]
@Jesus any plan for release?
